internal_testsuites/typescript/src/testsuite/startosis_package_test/startosis_package.test.ts

```diff
@@ -73,7 +73,7 @@ test("Test valid Starlark package with input - missing key in params", async ()
         }
 
         expect(runResult.value.interpretationError).not.toBeUndefined()
-        expect(runResult.value.interpretationError?.getErrorMessage()).toContain("Evaluation error: struct has no .greetings attribute")
+        expect(runResult.value.interpretationError?.getErrorMessage()).toContain("Evaluation error: key \"greetings\" not in dict")
```

Comment:
nit: Is there a way we can print the dict name here? We should have the line number, so should be fine, but in case it's easy

Reply:
I believe this comes from Starlark, we don't control the error message